screens/shoppingList

```diff
@@ -0,0 +1,286 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image } from 'react-native';
+import FontLoad from './components/FontLoad';
+
+export default class ShoppingList extends React.Component {
+
+
+  render() {
+    return (
+      this.state.fontLoaded ? (
+        <View style={styles.container}>
+           <View style={styles.rectangleOrange} />
+           <View style={styles.rectangleTopLightBlue} />
+           <View style={styles.rectangleMid} />
+           <View style={styles.rectangleBottomLightBlue} />
+           <Text style = {styles.header}>Shopping List</Text>
+           <Image source ={require('./assets/images/shopping-cart.png')} style={styles.ShoppingCart} />
+	         <Image source ={require('./assets/images/person.png')} style={styles.person} />
+           <Text style = {styles.TopBoxHeader}>Name</Text>
+           <Text style = {styles.TopBoxSubHeader}>Time, Location</Text>
+           <Text style = {styles.TopBoxSummary}>Shopping Cart Summary</Text>
+           <Text style = {styles.TopBoxETA}>ETA: 2 hours</Text>
+           <Text style = {styles.MidBoxHeader}>Name</Text>
+           <Text style = {styles.MidBoxSubHeader}>Time, Location</Text>
+           <Text style = {styles.MidBoxSummary}>Shopping Cart Summary</Text>
+           <Text style = {styles.MidBoxETA}>ETA: 2 hours</Text>
+           <Text style = {styles.BottomBoxHeader}>Name</Text>
+           <Text style = {styles.BottomBoxSubHeader}>Time, Location</Text>
+           <Text style = {styles.BottomBoxSummary}>Shopping Cart Summary</Text>
+           <Text style = {styles.BottomBoxETA}>ETA: 2 hours</Text>
+        </View>
+    ) : null
+    )
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  rectangleOrange: {
+    flex: 1,
+    backgroundColor: '#FF715B',
+    alignItems: 'center',
+    justifyContent: 'center',
+    position: 'absolute',
+    width: Dimensions.get("screen").width,
+    height: 320,
+    left: 0,
+    top: 0,
+    borderRadius: 2,
+  },
+  rectangleTopLightBlue: {
+    flex: 1,
+    backgroundColor: '#6DC4E0',
+    alignItems: 'center',
+    justifyContent: 'center',
+    position: 'absolute',
+    width: 345,
+    height: 134,
+    top: 190,
+  },
+  rectangleMid: {
+    flex: 1,
+    backgroundColor: '#605DF1',
+    alignItems: 'center',
+    justifyContent: 'center',
+    position: 'absolute',
+    width: 345,
+    height: 134,
+    top: 340,
+  },
+  rectangleBottomLightBlue: {
```

Comment:
This style name is too specific `rectangleBottomLightBlue`

This light blue container will most likely be used multiple times - try to give it a more generic name